src/centralserver/admin-ui/package.json

```diff
@@ -34,6 +34,7 @@
     "@apidevtools/swagger-cli": "~4.0.4",
     "@mdi/font": "~5.9.55",
     "@types/jest": "~26.0.8",
+    "@types/lodash": "^4.14.177",
```

Comment:
After closer study I can confirm that the local deepClone is NOT working, it's so much simpler than the lodash - alternative. 
Debugger shows, that deepClone produces pretty empty copy of Store object and specifically the `mutations` object (as well as `getters` & `actions`) is copied just as an empty object.  This is used only in test code, so no bloat to production build and I changed import so that it only imports this one function.

Reply:
Remedy proposal: 39fd70e